### PR TITLE
Fix scrapper and update data

### PR DESCRIPTION
**Pull request by RenzoMinelli**
### Summary
Update subject regex to reflect changes in bedelias as currently scrapper doesn't work. Also update scrapped data.

### CODE REVIEW

File: lib/scraper/bedelias.rb [hunk 1]
```diff
@@ -10,7 +10,8 @@ module Bedelias
 
     # 5265 - CIENCIAS HUMANAS Y SOCIALES - min: 10 créditos
     # SRN14 - MATEMÁTICA DISCRETA I - créditos: 10
-    CODE_NAME_CREDITS_REGEX = /\A(\w+) - (.+) - (?:min|créditos): (\d+)( créditos)?\z/
+    GROUP_CODE_NAME_CREDITS_REGEX = /\A(\w+) - (.+) - (?:min|créditos): (\d+)( créditos)?\z/
```
[the remark this answers is not in the document]
Comment (RenzoMinelli) [reply lib/scraper/bedelias.rb]: Updated in a148a28341ce711b8bbd4926674e40343f537316

File: lib/scraper/bedelias.rb [hunk 2]
```diff
@@ -10,7 +10,8 @@ module Bedelias
...
+    SUBJECT_CODE_NAME_CREDITS_REGEX = /\A(\w+) - (.+) - (?:min|créditos): (\d+)(?: .*)?\z/
```
[the remark this answers is not in the document]
Comment (RenzoMinelli) [reply lib/scraper/bedelias.rb]: Updated in a148a28341ce711b8bbd4926674e40343f537316

File: lib/scraper/bedelias.rb [hunk 3]
```diff
@@ -9,8 +9,9 @@ module Bedelias
     include Capybara::DSL
 
     # 5265 - CIENCIAS HUMANAS Y SOCIALES - min: 10 créditos
+    GROUP_CODE_NAME_CREDITS_REGEX = /\A(\w+) - (.+) - (?:min): (\d+)( créditos)?\z/
     # SRN14 - MATEMÁTICA DISCRETA I - créditos: 10
-    CODE_NAME_CREDITS_REGEX = /\A(\w+) - (.+) - (?:min|créditos): (\d+)( créditos)?\z/
+    SUBJECT_CODE_NAME_CREDITS_REGEX = /\A(\w+) - (.+) - (?:créditos): (\d+)(?: programa)?\z/
```
Comment (EmilioCristalli) [inline lib/scraper/bedelias.rb:14]: just to confirm, the `?:` in `(?: programa)?` is just to not make it part of the `captures` ?

Comment (RenzoMinelli) [reply lib/scraper/bedelias.rb:14]: Yeah exactly, we don't need to capture it in a group.

Comment (EmilioCristalli) [reply lib/scraper/bedelias.rb:14]: can we do the same for the group regex to keep it consistent? or remove from both?

Comment (RenzoMinelli) [reply lib/scraper/bedelias.rb:14]: Yeah totally. Updated in c8bb7c883d3effd6d12a533e3d9face263ea894b

File: lib/scraper/bedelias.rb [hunk 4]
```diff
@@ -9,8 +9,9 @@ module Bedelias
     include Capybara::DSL
 
     # 5265 - CIENCIAS HUMANAS Y SOCIALES - min: 10 créditos
+    GROUP_CODE_NAME_CREDITS_REGEX = /\A(\w+) - (.+) - (?:min): (\d+)( créditos)?\z/
```
[the remark this answers is not in the document]
Comment (RenzoMinelli) [reply lib/scraper/bedelias.rb]: Actually all groups must have it, here's a picture of all groups.


image

[the remark this answers is not in the document]
Comment (EmilioCristalli) [reply lib/scraper/bedelias.rb]: should we make it required then?

[the remark this answers is not in the document]
Comment (RenzoMinelli) [reply lib/scraper/bedelias.rb]: Updated in a38c7cb8ee93d5c082d3f5a8163a00ca3663012a

**Review by EmilioCristalli — APPROVED** [review]

**Pull request merged**